controller/src/main.rs

```diff
@@ -35,19 +35,9 @@ async fn main() {
     let state = AppState::new();
 
     let cors = CorsLayer::new()
-        .allow_origin([
-            "http://localhost:8880"
-                .parse::<axum::http::HeaderValue>()
-                .unwrap(),
-            "http://127.0.0.1:8880"
-                .parse::<axum::http::HeaderValue>()
-                .unwrap(),
-        ])
-        .allow_methods([axum::http::Method::POST, axum::http::Method::DELETE])
-        .allow_headers([
-            axum::http::HeaderName::from_static("hx-request"),
-            axum::http::HeaderName::from_static("hx-current-url"),
-        ]);
+        .allow_origin(Any)
+        .allow_methods(Any)
+        .allow_headers(Any);
```

Comment:
The main thing here is to allow all origins, but maybe granular control over headers and methods may be good.